gems/pending/appliance_console.rb

```diff
@@ -28,7 +28,8 @@
 include HighLine::SystemExtensions
 
 require 'i18n'
-LOCALES = File.join(File.expand_path(File.dirname(__FILE__)), "appliance_console/locales/*.yml")
+locales_dir = ENV['CONTAINER'].to_s.casecmp("true") == 0 ? "container" : "appliance"
+LOCALES = File.join(File.expand_path(File.dirname(__FILE__)), "appliance_console/locales/#{locales_dir}/*.yml")
```

Comment:
Prefer `__dir__` over `File.dirname(__FILE__)`.  No need to necessarily change for this PR, as it was like that before, but for future reference.